### ISSUE TITLE
About the checkpoint file

### ISSUE THREAD

**Issue by MHpurity** [ningnbai/PIM-Net#2]
Dear author,
  Could you like to share the  checkpoint file of your model with me?

**Comment by ningnbai** [comment]
Hi! We sincerely apologize for the inconvenience. During the manuscript preparation, we unified the naming of several modules, which resulted in discrepancies between the module names in the uploaded repository and those used in our locally trained .pth files. To ensure reproducibility, we recommend training the model directly using the uploaded code; its training pipeline is consistent with the settings described in the paper, and the final performance closely matches the reported results.
If you encounter any issues during model training or execution, we would be more than happy to provide further assistance.

**Comment by MHpurity** [comment]
> Hi! We sincerely apologize for the inconvenience. During the manuscript preparation, we unified the naming of several modules, which resulted in discrepancies between the module names in the uploaded repository and those used in our locally trained .pth files. To ensure reproducibility, we recommend training the model directly using the uploaded code; its training pipeline is consistent with the settings described in the paper, and the final performance closely matches the reported results. If you encounter any issues during model training or execution, we would be more than happy to provide further assistance.

Hi! Thanks for your reply!
I have some question about the training code. I was trying to locate the code for the fine-tuning stage, but I couldn't find it. Is the existing train.py file just used to training  the backbone?

**Comment by ningnbai** [comment]
Hi! train.py is used for training and fine-tuning the model. During the fine-tuning phase, it is necessary to load the pre-trained .pth weights.

**Comment by MHpurity** [comment]
> Hi! train.py is used for training and fine-tuning the model. During the fine-tuning phase, it is necessary to load the pre-trained .pth weights.

Thanks for your reply! Every success in your job!

**Comment by MHpurity** [comment]
Dear author,
Sorry to bother you again. Should I freeze the backbone weights after loading the pre-trained .pth file, or should I just allow them both to be trainable? This detail was not covered in the paper.
Are  the training parameters of the fine-tune (eg. bathsize, learning rate,) the same with the pre-train stage?

**Comment by ningnbai** [comment]
Hi! Sorry for the lack of clarity in the paper. There are various approaches to fine-tuning, and my method is to first load the pre-trained .pth file, then proceed with the fine-tuning training without freezing any parameters.

**Comment by MHpurity** [comment]
> Hi! Sorry for the lack of clarity in the paper. There are various approaches to fine-tuning, and my method is to first load the pre-trained .pth file, then proceed with the fine-tuning training without freezing any parameters.

Thanks for your kind reply.

**Comment by MHpurity** [comment]
Sorry to bother you again. 
I test the trained.pth file. And my results were significantly different:
    eg: CASIA1   Average F1 0.4599 (yours: 0.569)
I suspect the discrepancy may stem from an error or misunderstanding in my implementation of the dataset processing for either the pre-training or fine-tuning phase. My pre-training dataset consists of 27,639 images randomly taken from the spliced_nist.zip and dresden_spliced_mscoco.zip parts of the Synthetic Data (TIP2019). The fine-tuning dataset was constructed using the specific datasets referenced in the table presented in your paper.
Could you please clarify the specific details of training data loading and preprocessing for both the pre-train and fine-tune stages?

**Comment by ningnbai** [comment]
My apologies for the delayed reply. It is recommended that you set a slightly smaller learning rate during the fine-tuning phase and train for about 10 epochs.

**Comment by MHpurity** [comment]
> My apologies for the delayed reply. It is recommended that you set a slightly smaller learning rate during the fine-tuning phase and train for about 10 epochs.

Dear author,
Thanks for your reply. Can you provide me with the dresden_spliced_mscoco.zip? I can‘t successfully download the complete dresden_spliced_mscoco.zip file from the shared link you provided.
